extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/conf/CASGuacamoleProperties.java

```diff
@@ -68,5 +69,30 @@ private CASGuacamoleProperties() {}
         public String getName() { return "cas-clearpass-key"; }
 
     };
+  
+    /**
+     * The attribute used for group membership
+     * example:  memberOf  (case sensitive)
+     */
+    public static final StringGuacamoleProperty CAS_GROUP_ATTRIBUTE =
+            new StringGuacamoleProperty() {
+
+        @Override
+        public String getName() { return "cas-group-attribute"; }
+
+    };
 
+    /**
+     * The attribute used for group DN Format
+     * such as CN=%s,OU=myou,DC=example,DC=com  (case sensitive)
+     * used to strip all but %s.  This is only necessary when
+     * CAS backend is LDAP. 
+     */
```

Comment:
I'm not sure this is correct. How is the format an attribute?

Reply:
Modified the comment in latest commits to clarify this. Thanks for identifying this unclear comment.